lib/sycamore/sycamore/transforms/partition.py

```diff
@@ -476,101 +488,48 @@ def partition(self, document: Document) -> Document:
                 extract_images=self._extract_images,
                 batch_size=self._batch_size,
                 batch_at_a_time=self._batch_at_a_time,
+                local=self._local,
+                aryn_api_key=self._aryn_api_key,
+                aryn_partitioner_address=DEFAULT_ARYN_PARTITIONER_ADDRESS,
                 use_cache=self._use_cache,
             )
         except Exception as e:
             path = document.properties["path"]
             raise RuntimeError(f"SycamorePartitioner Error processing {path}") from e
 
-        elements = []
-        for i, r in enumerate(result):
-            for ele in r:
-                ele.properties["page_number"] = i + 1
-                elements.append(ele)
-
         document.elements = elements
         document = reorder_elements(document, self._elements_reorder)
         return document
 
 
-class ArynPartitioner(Partitioner):
-    """
-    The ArynPartitioner runs a SycamorePartitioner on Aryn's GPU servers to offload compute.
-
-    Args:
-        aryn_token: The account token used to authenticate with Aryn's servers.
-        threshold: The threshold to use for accepting the models predicted bounding boxes. A lower
-             value will include more objects, but may have overlaps, a higher value will reduce the
-             number of overlaps, but may miss legitimate objects.
-        use_ocr: Whether to use OCR to extract text from the PDF. If false, we will attempt to extract
-             the text from the underlying PDF.
-        ocr_images: If set with use_ocr, will attempt to OCR regions of the document identified as images.
-        ocr_tables: If set with use_ocr, will attempt to OCR regions on the document identified as tables.
-             Should not be set when `extract_table_structure` is true.
-        extract_table_structure: If true, runs a separate table extraction model to extract cells from
-             regions of the document identified as tables.
-        extract_images: If true, crops each region identified as an image and attaches it to the associated
-             ImageElement. This can later be fed into the SummarizeImages transform.
-
-    Example:
-        The following shows an example of using the ArynPartitioner to partition a PDF and extract
-        both table structure and images.
-
-        .. code-block:: python
-
-            context = sycamore.init()
-            partitioner = ArynPartitioner(extract_table_structure=True, extract_images=True)
-            context.read.binary(paths, binary_format="pdf")\
-                .partition(partitioner=partitioner)
-    """
-
+class SycamorePartitioner(ArynPartitioner):
```

Comment:
If this is for compatibility reasons (I think we have notebooks around) it's fine to keep this but let's document that. Delete otherwise